File: app/src/main/java/dk/kea/androidgame/martin/myfirstgameengine/touch/TouchEventPool.java
package dk.kea.androidgame.martin.myfirstgameengine.touch;

import dk.kea.androidgame.martin.myfirstgameengine.core.Pool;

public class TouchEventPool extends Pool<TouchEvent>
{
    @Override
    protected TouchEvent newItem()
    {
        return new TouchEvent();
    }
}

File: app/src/main/java/dk/kea/androidgame/martin/myfirstgameengine/touch/TouchEventPool.kt
package dk.kea.androidgame.martin.myfirstgameengine.touch

import dk.kea.androidgame.martin.myfirstgameengine.core.Pool

class TouchEventPool : Pool<TouchEvent>() {
    override fun newItem(): TouchEvent {
        return TouchEvent()
    }
}
